testing/citest/spinnaker_testing/spinnaker.py

```diff
@@ -358,13 +366,13 @@ def new_instance_from_bindings(cls, name, status_factory, bindings, port):
     """
     host_platform = cls.__determine_host_platform(bindings)
     if host_platform == 'native':
-      base_url = (None if not bindings['NATIVE_HOSTNAME']
-                  else 'http://{host}:{port}'.format(
-                      host=bindings['NATIVE_HOSTNAME'],
-                      port=bindings['NATIVE_PORT'] or port))
-
+      base_url = None
+      if bindings['NATIVE_HOSTNAME']:
```

Comment:
why would the hostname include the protocol? That isnt a hostname. Is that a real thing?
Why not add a NATIVE_PROTOCOL binding to indicate http vs https?